net/fwknop/files/fwknopd.init

```diff
@@ -55,7 +55,7 @@ generate_configuration()
 	local PCAP_INTF=
 	local USER_CONFIG_PATH=/etc/fwknop/fwknopd.conf
 	local DEFAULT_UCI_NETWORK=wan
-	local DEFAULT_FWKNOPD_IFNAME=eth0
+	local DEFAULT_FWKNOPD_IFNAME=$(uci show network.$DEFAULT_UCI_NETWORK.ifname | sed -n "s/.*\'\([^ ]*\)\'/\1/p")
```

Comment:
Please consider using `network_get_device` from `/lib/functions/network.sh` for this.